ethereum/gateway-eth-ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@identity.com/gateway-eth-ts",
-  "version": "0.1.0-beta.1",
+  "version": "0.1.1",
```

Comment:
I could not publish 0.1.0 because of the following error: Cannot publish over previously published version \"0.1.0\".". I think this package might have been published and then removed from NPM.

"Even if a package version is unpublished, that specific name and version combination can never be reused. In order to publish the package again, a new version number must be used."